js/kucoinfutures.js

```diff
@@ -1256,14 +1253,9 @@ module.exports = class kucoinfutures extends kucoin {
         await this.loadMarkets ();
         // only fetches one balance at a time
         const request = {};
-        const coin = this.safeString (params, 'coin');
-        let defaultCode = this.safeString (this.options, 'code');
-        const fetchBalanceOptions = this.safeValue (this.options, 'fetchBalance', {});
-        defaultCode = this.safeString (fetchBalanceOptions, 'code', defaultCode);
-        const code = this.safeString (params, 'code', defaultCode);
-        if (coin !== undefined) {
-            request['currency'] = coin;
-        } else if (code !== undefined) {
+        const defaultCode = this.safeString (this.options, 'defaultCode');
+        const code = this.safeString2 (params, 'code', 'currency', defaultCode);
+        if (code !== undefined) {
```

Comment:
We used to support both the exchange-specific param for the currency id and the unified currency code separately cause we should not mix them.